testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/configs/ConfigDefault.java

```diff
@@ -74,15 +80,23 @@ public String getConfigName()
                 "tpch_connector");
     }
 
+    @Override
+    public String getJavaHome()
+    {
+        return "/usr/lib/jvm/zulu-11";
```

Comment:
It can be Config's attribute, but I would consider modelling this as a EnvironmentOptions attribute.

@wendigo  @losipiuk @hashhar wdyt?

Reply:
You would like to pass path there? Version number? I'm not fully convinced if this is the right place to add that. My plan was to create separate environment configs for JDK17 (ConfigCdh5Jdk17)

Reply:
EnvOptions are useful if you want to run tests with different options as part of same suite - which I don't think is possible since the tests are already ~1h long.

I think adding new Config is a better option here.

Reply:
Our `Config` classes should be thought as `HadoopConfig` or `HiveConfig`.
They are designed to effectively create matrix of hive tests.

One day we will run all tests  on Java 17. Having a parameter to `env up` / `test run` should be sufficient for transition period.
The advantage of modelling this as Config is that it looks better in ci.yml. Is this the reason?

Reply:
Ah, sorry I confused your suggestion with ExtraOptions. Yeah, EnvironmentOptions seems better than Config - we'll eventually get rid of the JDK11 tests.

Reply:
@hashhar @wendigo if we go with EnvironmentOptions, do we have a way to support this in ci.yml?

eg add `--trino-jdk-version` to `EnvironmentOptions`
and then add `--trino-jdk-version`  to `SuiteRunOptions` as a pass-through.

Reply:
Done. PTAL now.